test/ce-currentFragment/test-14.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<script>
+  'use strict';
+  window.test14 = async_test('Fixture 14');
```

Comment:
This __async_test__ is necessary. Otherwise, when you want to test third level of nesting, testharness will stop working.